source/api_documentation/network_integration/advertiser_campaigns/index.rst

```diff
@@ -296,6 +296,10 @@ Node Parameters and Usage
     - prompt
     - Prompt the caller to verify the guessed location or confirm through input. Useful if geographical data is important or useful in a condition node.
 
+  * - DynamicRoute
```

Comment:
This is kinda an odd layout for these docs, since it lists the node type and param in more than one place. I think we should have a `(read-only)` in each spot, so right here and also down in the `Parameter Details` table.

Reply:
Ack, updated in both places. Also fixed up the inaccurate column header, changed to "property" to match all the other API tables.

![image](https://user-images.githubusercontent.com/1828707/40866711-214ebfe0-65b4-11e8-8493-34c1d8a6f087.png)